Volume_1/Book_about_QuadratizationNotes.bib

```diff
@@ -1,2 +1,2 @@
-@CONTROL{REVTEX41Control}
-@CONTROL{apsrev41Control,author="00",editor="1",pages="1",title="0",year="0"}
+@CONTROL{REVTEX42Control}
+@CONTROL{apsrev42Control,author="08",editor="1",pages="0",title="0",year="1"}
```

Comment:
Why did you have to change author from 00 to 08, and pages from 1 to 0 and year from 0 to 1?

Reply:
yeah again, it's just weird compilation locally, maybe some dependencies are different? I'll revert it!